jekyll/_cci2/enable-checks.md

```diff
@@ -25,16 +25,17 @@ To use the CircleCI Check integration, you first need to navigate to the Org Set
 
 ### Prerequisites
 
-- You must be using the cloud-hosted version of Circle CI.
+- You must be using the cloud-hosted version of CircleCI.
 - Your project must be using CircleCI 2.0 with [Workflows]( {{ site.baseurl }}/2.0/workflows/).
 - You must be an Admin on your GitHub repository to authorize installation of the CircleCI Checks integration.
 
 ### Steps
 
-1. Click the Settings tab in the CircleCI app main menu.
+1. In the CircleCI sidebar, select "Organization Settings"
```

Comment:
I reckon we should decide (and put in the style guide if it isn't already) how to refer to specific pages in the UI - bold/quotation marks/literal text. Also for buttons. We do different things all the time. Maybe I'll post something in docs channel about it.